lib/mail/parsers/rfc_2822.ex

```diff
@@ -328,7 +328,7 @@ defmodule Mail.Parsers.RFC2822 do
         decoded_string =
           case String.upcase(encoding) do
             "Q" ->
-              Mail.Encoders.QuotedPrintable.decode(encoded_string)
+              Mail.Encoders.QuotedPrintable.decode(String.replace(encoded_string, "_", " "))
```

Comment:
why not handle this directly in the `QuotedPrintable` encoder?

Reply:
Because it’s not a part of the quoted printable spec, it’s a part of the encoded word spec. In quoted printable, you would use either a space  or =20

Reply:
Ah ok, I'm pretty far removed from the spec review at this point. I'll trust your judgement on this. This looks like the correct implementation then